From 60da77d82287e302d00d523b5bef9803db7a273d Mon Sep 17 00:00:00 2001
From: Carlos Soriano <csoriano@gnome.org>
Date: Wed, 29 Apr 2015 16:46:08 +0200
Subject: window: fix style tag outside <object>

The style tag needs to be inside the <object> scope

diff --git a/src/nautilus-window.ui b/src/nautilus-window.ui
index 47100ad..105fe2a 100644
--- a/src/nautilus-window.ui
+++ b/src/nautilus-window.ui
@@ -89,10 +89,10 @@
                                 <property name="receives_default">True</property>
                                 <property name="no_show_all">True</property>
                                 <property name="margin_end">6</property>
+                                <style>
+                                  <class name="text-button"/>
+                                </style>
                               </object>
-                              <style>
-                                <class name="text-button"/>
-                              </style>
                             </child>
                             <child>
                               <object class="GtkButton" id="notification_delete_close">
-- 
cgit v0.10.2

